packages/storybook/src/stories/playground/RecipeReviewCard.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
The correct import path

```suggestion
import * as React from 'react';
```

Reply:
copied from MUI doc :D

Reply:
from v4, not v5 docs :p